enhancements/management-workload-partitioning.md

```diff
@@ -655,7 +655,7 @@ None
 ### Test Plan
 
 We will add a CI job to ensure that all release payload workloads and
-their namespaces have the `workload.openshift.io/management`
+their namespaces have the `target.workload.openshift.io/management`
```

Comment:
This is only the pod annotation, the expected namespace annotations is different.

Reply:
Thanks, fixed.